R/get-net-task.R

```diff
@@ -20,9 +32,14 @@
 #' @return returns an instance of of the specified `.NET` task.
 #'
 #' @keywords internal
-.getNetTaskFromCache <- function(taskName) {
+.getNetTaskFromCache <- function(taskName, isMoBiR = FALSE) {
```

Comment:
@rwmcintosh See, here, we are only calling to .NET if the task has not been created, yet. But `.getNetTask()` always calls to `OSPSuite.R.Api`, so I had to create a separate function to call to MoBi.R. Not nice.

Reply:
Well, yeah, but this needs to be somehow addressed. It can be hidden by a .NET API in this project, or in R. Someone has to know where to find this method.

Reply:
It feels to me that this should be TWO methods not one
.getMoBiTaskFromCache 
.getCoreTaskFromCache

this isMoBiR as aparameter is not ergonomic at all